packages/storefront-events-collector/src/utils/aep/productListItems.ts

```diff
@@ -35,7 +35,7 @@ const createProductListItems = (
                 quantity: productListItemFromCustomContext?.quantity || Number(item.quantity),
                 priceTotal:
                     productListItemFromCustomContext?.priceTotal ||
-                    (Number(item.pricing?.regularPrice) || 0) * Number(item.quantity),
+                    Number(((Number(item.pricing?.regularPrice) || 0) * Number(item.quantity)).toFixed(2)),
```

Comment:
I feel like a lot of context is lost here. I would move this parsing logic out into a function to make it more testable and then apply that function to any number values that need it.